urls.py

```diff
@@ -92,6 +90,11 @@
     url(r'^submit_feedback/$', submit_feedback, name='submit_feedback'),
 ]
 
+for plugin_url_modules in plugins.get_contributions("urls:domain_specific"):
+    for module in plugin_url_modules:
+        domain_specific.append(url(r'^', include(module)))
```

Comment:
the alternative is to put these in locasettings: https://github.com/dimagi/commcare-hq/pull/28094/files#diff-de6dd4b4c889fe0882cfd3f6df5aa451R92